src/QsFmt/Formatter/SyntaxTree/Statement.fsi

```diff
@@ -141,75 +175,65 @@ type internal Let =
     }
 
 /// <summary>
-/// A <c>return</c> statement.
+/// An <c>update</c> statement.
```

Comment:
This could use a bit more detail; the language docs do not have a section about an update statement. It would be helpful if the doc comments in the code here use terminology that facilitates finding the corresponding section in the language docs. If we wanted to *really* nice, we could even link to the language docs...
Could you please clarify how an update statement differs from a SetWith?

To align with language docs, both SetWith and UpdateStatement are referred to as "evaluate-and-reassign statements" in the language docs: https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/2_Statements/VariableDeclarationsAndReassignments.md#evaluate-and-reassign-statements.

Reply:
I mentioned the evaluate-and-reassign name and gave an example for the Update statements.